src/sys/led.cpp

```diff
@@ -0,0 +1,98 @@
+#include "led.hpp"
+
+#include "Particle.h"
+
+Timer SFLed::ledTimer(SF_LED_BLINK_MS, SFLed::doLEDs, false);
```

Comment:
I don't like that this lives here - it becomes very difficult to tell what is happening because this doesn't live with the other timers in `system.cpp`.